examples/benchmark/ocis/Makefile

```
@@ -0,0 +1,7 @@
+docker-compose-up:
+	bash create-identifier-registration.sh
+	docker-compose up
```

```suggestion
	docker-compose up -d
```

We want to normally put the docker processes and output into the background. That gives back the terminal when you don't really care about seeing the output to debug stuff.

So we could also have another target:
```
docker-compose-up-foreground:
	bash create-identifier-registration.sh
	docker-compose up
```